M2/Macaulay2/d/interp.dd

```diff
@@ -662,6 +662,13 @@ runBasicTests(e:Expr):Expr := (
      else WrongNumArgs(0));
 setupfun("runBasicTests",runBasicTests);
 
+getHistory(e:Expr):Expr := (
+    when e is n:ZZcell do if isInt(n)
+    then toExpr(tostring(getHistory(toInt(n))))
+    else WrongArgSmallInteger()
+    else WrongArgZZ());
+setupfun("getHistory", getHistory);
```

Comment:
Thank you for renaming this!  I've regretted my decision to call it `historyGet` ever since I added it.  I think I was just trying to mimic `history_get`, but `getHistory` makes way more sense.

Reply:
readline developers had no sense of style ...